### PR TITLE
Chromatic ignore flaky images

### PR DESCRIPTION
**Pull request by domlander**
## What does this change?

When running a chromatic build, ignore images prone to causing failures.

Ignores lazy-loaded images in Cards, which are also prone to causing Chromatic failures.

## Why?

Rendering of images are non-deterministic, which can cause Chromatic build failures. Slight image differences caused by  the browser rendering engine is not behaviour we want Chromatic to flag. For example, in the image in this build failure, there are only a few pixels of difference in the entire image and this has caused the build to fail: https://www.chromatic.com/test?appId=637e406971a9af18ddba0505&id=660d43a8ddbe4a74d065bebf

Lazy-loaded images in cards are not always rendered in a Chromatic snapshot: https://www.chromatic.com/test?appId=63e251470cfbe61776b0ef19&id=660ece0c32ef7ef15789c65c

### CODE REVIEW

File: apps-rendering/src/components/editions/layout/layout.stories.tsx
Comment (mxdvl) [inline apps-rendering/src/components/editions/layout/layout.stories.tsx:1]: Is this simply a replacement of the deprecated import?

**Review by mxdvl — APPROVED** [review]
Hopefully this helps with false positives.

---

Some thoughts for a follow-up: would you mind adding some docs to the `data-chromatic` attribute [in the JSX namespace](https://github.com/guardian/dotcom-rendering/blob/9e01b8b9f14719e1930ea1400950c4114563ba31/dotcom-rendering/index.d.ts#L419)? This will help developers understand what it does and could link to the Chromatic docs!

**Review by abeddow91 — APPROVED** [review]

File: apps-rendering/src/components/editions/layout/layout.stories.tsx
Comment (domlander) [reply apps-rendering/src/components/editions/layout/layout.stories.tsx:1]: Yes

File: dotcom-rendering/index.d.ts [hunk 1]
```diff
@@ -448,6 +448,15 @@ declare namespace JSX {
...
 		'data-link-name'?: string;
+		/**
+		 * Ignore a DOM element in Chromatic builds with `data-chromatic="ignore"`.
+		 *
+		 * https://www.chromatic.com/docs/ignoring-elements/#ignore-dom-elements
+		 *
+		 * Note that if the dimensions of the ignored element
+		 * change, Chromatic will still capture the incoming changes.
+		 */
+		'data-chromatic'?: string;
```
Comment (mxdvl) [inline dotcom-rendering/index.d.ts:459]:
```suggestion
		'data-chromatic'?: "ignore";
```

Comment (domlander) [reply dotcom-rendering/index.d.ts]: Whoops. Thanks Max!

**Pull request merged**